starter-code/pizza.js

```diff
@@ -1 +1,174 @@
+  //   if (totalPrice<10) {
+  //     document.getElementById('finalPrice').innerHTML=(basePrice);
+  // } else {
+  //     document.getElementById('finalPrice').innerHTML=(totalPrice);
+  // }
+// }});
+
+
+  //  if ($(".btn-pepperonni").hasClass('active')) {
+  //    $('#pepper').show();
+  //    currentPrice=basePrice+1;
+  //    document.getElementById('finalPrice').innerHTML=(currentPrice);
+  //    $('finalPrice').innerHTML().replaceWith($(currentPrice));
+  //
+  // } else {$('#pepper').hide();
+  //    newPrice=currentPrice-1;
+  //    document.getElementById('finalPrice').innerHTML=(newPrice);
+  //    }
+
+// MUSHROOM button
+
+$(".btn-mushrooms").click(function() {
+  $(".mushroom").toggle();
+  $("#shroom").toggle();
+  $(this).toggleClass("active");
+  calculatePrice();});
+  // if ($(this).hasClass('active')) {
+  //   totalPrice=Number(totalPrice) + 1;
+  //   document.getElementById('finalPrice').innerHTML=(totalPrice);
+  // } else {
+  //   totalPrice=Number(totalPrice) - 1;
+  //   if (totalPrice<10) {
+  //     document.getElementById('finalPrice').innerHTML=(basePrice);
+  // } else {
+  //     document.getElementById('finalPrice').innerHTML=(totalPrice);
+  // }}});
+//   if ($(".btn-mushrooms").hasClass('active')) {
+//     $('#shroom').show();
+//     currentPrice=basePrice+1;
+//     document.getElementById('finalPrice').innerHTML=(currentPrice);
+//
+//  } else {$('#shroom').hide();
+//    newPrice=currentPrice-1;
+//    document.getElementById('finalPrice').innerHTML=(newPrice);}
+// }
+// );
+
+$(".btn-green-peppers").click(function() {
+  $(".green-pepper").toggle();
+  $(this).toggleClass("active");
+  $("#greenpep").toggle();
+  calculatePrice();});
+//   if ($(".btn-green-peppers").hasClass('active')) {
+//     $('#greenpep').show();
+//     currentPrice=currentPrice+1;
+//     document.getElementById('finalPrice').innerHTML=(currentPrice);
+//     $('finalPrice').innerHTML().replaceWith($(currentPrice));
+//  } else {$('#greenpep').hide();}
+//
+//  newPrice=currentPrice-1;
+//  document.getElementById('finalPrice').innerHTML=(newPrice);}
+// );
+
+ $('.btn-sauce').click(function() {
+  $('#whitesauce').toggle();
+  $('.sauce-white').toggle();
+  $(this).toggleClass("active");
+  calculatePrice();}
+//   if ($(this).hasClass('active')) {
+//     totalPrice=Number(totalPrice) + 3;
+//   } else {
+//     totalPrice=Number(totalPrice) - 3;
+//   }
+// });
+//   if ($(".btn-sauce").hasClass('active')) {
+//     $('#whitesauce').show();
+//     currentPrice=currentPrice+3;
+//     document.getElementById('finalPrice').innerHTML=(currentPrice);
+//     $('finalPrice').innerHTML().replaceWith($(currentPrice));
+//  } else {$('#whitesauce').hide();}
+//   newPrice=currentPrice-3;
+//   if (newPrice<=10) {
+//   document.getElementById('finalPrice').innerHTML=(basePrice);}
+// else {document.getElementById('finalPrice').innerHTML=(currentPrice);}});
+);
+$('.btn-crust').click(function() {
+  $('.crust').toggle();
```

Comment:
Here, you're actually removing the crust so the pepperonis are just floating in the air.  If instead of toggling the crust, you can remove the `crust-gluten-free` class and it will look like just a normal crust.  This one is slightly different than the rest.